lib/bolt/transport/docker/connection.rb

```diff
@@ -8,21 +8,65 @@ module Transport
     class Docker < Base
       class Connection
         def initialize(target)
-          # lazy-load expensive gem code
-          require 'docker'
```

Comment:
I think this means it can be removed from the `gemspec` / `Gemfile` as well

Reply:
Hopefully yes.  Baby steps first 😃

Reply:
The door is now closed `docker-api`

![closing the door](https://gifrific.com/wp-content/uploads/2015/03/Frank-Underwood-Closing-Door-House-of-Cards.gif)

Reply:
I seem to have broken some tests as it's looking for `Docker::` :-)